pithos/pandora/pandora.py

```diff
@@ -371,6 +373,28 @@ def add_feedback(self, trackToken, rating):
     def delete_feedback(self, stationToken, feedbackId):
         self.json_call('station.deleteFeedback', {'feedbackId': feedbackId, 'stationToken': stationToken})
 
+    def get_ad_metadata(self, adTokens):
+        logging.info('attemping to retrieve ad metadata')
+        metadata = self.json_call('ad.getAdMetadata', {'adToken': adTokens,
+                                                       'supportAudioAds': True,
+                                                       'includeBannerAd': False,
+                                                       'includeTrackLength': True, 
+                                                       'additionalAudioUrl': 'HTTP_32_AACPLUS,HTTP_128_MP3',
+                                                       'returnAdTrackingTokens': True})
+
+        #Pandora at times returns metadata containing only an empty list of adTrackingTokens.
```

Comment:
__[flake8]__
* [E265] block comment should start with '# '

<!-- Comment by [SideCI](https://sideci.com) -->